preparing-to-code/terminal/README.md

```diff
@@ -33,18 +37,21 @@ When you are writing code, it is good to have a folder structure which is logica
 1. Verify that the folder structure was created appropriately by running the `pwd` command, the output should include your home directory and end with `/ada/jumpstart`.
```

Comment:
```suggestion
1. Verify that the folder structure was created appropriately by running the `pwd` command, the output should include your home directory and end with `/ada/build`.
```

Reply:
another change from jumpstart to build, but maybe this is already done in another PR.